### ISSUE TITLE
ForecasterRnn loss

### ISSUE THREAD

**Issue by AVPokrovsky** [JoaquinAmatRodrigo/skforecast#719]
Hi, can I create custom function for loss?

model = create_and_compile_model(
    series=data_train,
    levels=levels, 
    lags=lags,
    steps=steps,
    recurrent_layer="LSTM",
    recurrent_units=[100, 50],
    dense_units=[64, 32],
    optimizer=Adam(learning_rate=0.01), 
    loss=MeanSquaredError()

**Comment by AVPokrovsky** [comment]
https://keras.io/api/losses/#creating-custom-losses

**Comment by AVPokrovsky** [comment]
But only for keras 3 that does not work

**Comment by JoaquinAmatRodrigo** [comment]
Hi @AVPokrovsky,
You are right, skforecast is not yet compatible with Keras 3. We expect to add this compatibility in the next release.

@FernandoCarazoMelo

**Comment by fernando-carazo** [comment]
Hi @AVPokrovsky. Custom loss functions are not yet implemented in the create_and_compile_model function. You can add them if you use custom NNs.

**Comment by AVPokrovsky** [comment]
Ok. Thanks.
What is custom NNs?

**Comment by fernando-carazo** [comment]
A neural network that you create directly using PyTorch instead of using the skforecast utils function create_and_compile_model.

**Comment by fernando-carazo** [comment]
What you can do in the current version of skforecast is add any other built-in functions available in Keras
https://keras.io/api/losses/

**Comment by AVPokrovsky** [comment]
Ок

**Comment by AVPokrovsky** [comment]
Yes, I already tried it. Thanks.

**Comment by AVPokrovsky** [comment]
Hi, Is custom loss function still not available for the last release?

**Comment by FernandoCarazoMelo** [comment]
Hi @AVPokrovsky,

Thanks for keeping track of this thread!

Unfortunately, this feature is not yet available. We are currently
implementing other features with higher priority, but we keep your
suggestion on our backlog.

Best,
Fernando
